File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex25.java
//Importando Scanner
import java.util.Scanner;

public class Ex25 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Gratificação Natalina\n\n");
		
		//vars
		float hExtra, hFalta, H;
		
		//Entrada
		System.out.print("Insira a quantidade de horas extras trabalhadas: ");
		hExtra = ent.nextFloat();
		
		System.out.print("Insira a quantidade de horas-falta trabalhadas: ");
		hFalta = ent.nextFloat();
		
		//Processamento
		H = hExtra - ( 2*hFalta/3 );
		
		if ( H < 600 )
			System.out.println("Gratificação: R$ 100,00");
		else if ( H < 1200 )
			System.out.println("Gratificação: R$ 200,00");
		else if ( H < 1800 )
			System.out.println("Gratificação: R$ 300,00");
		else if ( H < 2400 )
			System.out.println("Gratificação: R$ 400,00");
		else
			System.out.println("Gratificação: R$ 500,00");
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex25.c
#include <stdio.h>

main()
{
    printf("Gratificacao Natalina\n\n");

    //vars
    float h, horasE, horasF;

    //Entradas
    printf("Insira a quantidade de horas extras trabalhadas: ");
    scanf("%f",&horasE);

    printf("Insira a quantidade de horas faltadas: ");
    scanf("%f",&horasF);

    //Processamento
    h = horasE - (2 * horasF) / 3;

    //Saída
    if ( h < 600 )
        printf("Gratificacao: R$ 100,00");
    else if ( h < 1200 )
        printf("Gratificacao: R$ 200,00");
    else if ( h <= 1800 )
        printf("Gratificacao: R$ 300,00");
    else if ( h < 2400 )
        printf("Gratificacao: R$ 400,00");
    else
        printf("Gratificacao: R$ 500,00");
}
